lib/tasks/populate_migrating_status.rake

```diff
@@ -0,0 +1,23 @@
+# Populate :migrating_status with data from :migrating
+# after this column has been added to orgs, users, and repos
+desc "Populate migrating_status for orgs, users, repos"
+task :populate_migrating_status do
+  require "travis"
+  require "travis/api/v3"
+  Travis::Database.connect
+
+  puts "Going to migrate status of #{repos.count} repos"
+  repos = Travis::API::V3::Models::Repository.where.not(migrated_at: nil)
+  repos.update_all(:migrating_status, :migrated)
```

Comment:
Do you know if this will run as a single query? If not, do you mind checking it?

Reply:
Actually, nvm:

```
travis-api-production::DATABASE=> select count(*) from repositories where migrated_at is not null;
 count 
-------
     1
(1 row)
```

Which also makes me wonder if we even need to run the script at all?

Reply:
This is strange because this means the `migrated_at` field is not being used at all right now. I'll dig in a bit more.

But sure, if this field is not in use, then we needn't run this rake task at all. Good catch, thanks

Reply:
It is being used by the merge app currently. The reason we don't see it that often in production is that we haven't migrated that many records.